packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/gestures.dart' show DragStartBehavior;
+import 'package:flutter/widgets.dart' show debugCheckHasMediaQuery;
```

Comment:
Since we're in widgets, you need to import the exact file

Reply:
ok ya, I seeeeee